developer/frameworks/articles/05-developing-fragments/reference-fragment-specific-tags.markdown

```diff
@@ -0,0 +1,109 @@
+# Fragment Specific Tags [](id=fragment-specific-tags)
+
+The following is a list of Fragment Specific tags in Liferay and their usage.
+
+## Making Text Editable [](id=making-text-editable)
+
+You can make any text of a fragment editable by enclosing it in an 
+`<lfr-editable>` tag like this:
+    
+    <lfr-editable id="unique-id" type="text">
+       This is editable text!    
+    </lfr-editable>
+ 
+The `lfr-editable` tag doesn't render without a unique `id`. 
+
+The following three `type` options are available inside of of `lfr-editable` tag:
+
+`text`: Creates a space for plain text that can be edited before publishing. 
+
+`image`: Must contain a valid `<img>` tag which can then be replaced with any
+image before publishing---including those from Documents and Media.
+ 
+`rich-text`: Provides rich text formatting, such as bold, italics, underline,
+links, and predefined styles.
+
```

Comment:
as mentioned before, we now have type 'link', and they can be visualized as buttons (primary or secondary) or link